src/broadcast/mpris/mprisplayer.cpp

```diff
@@ -0,0 +1,455 @@
+#include "broadcast/mpris/mprisplayer.h"
+
+#include <QCryptographicHash>
+#include <QtGlobal>
+
+#include "library/autodj/autodjprocessor.h"
+#include "library/coverartcache.h"
+#include "mixer/deck.h"
+#include "mixer/playerinfo.h"
+#include "mixer/playermanager.h"
+#include "moc_mprisplayer.cpp"
+
+namespace {
+
+const QString kPlaybackStatusPlaying = QStringLiteral("Playing");
+const QString kPlaybackStatusPaused = QStringLiteral("Paused");
+const QString kPlaybackStatusStopped = QStringLiteral("Stopped");
+
+// the playback will stop when there are no more tracks to play
+const QString kLoopStatusNone = QStringLiteral("None");
+// The current track will start again from the begining once it has finished playing
+const QString kLoopStatusTrack = QStringLiteral("Track");
+// The playback loops through a list of tracks
+const QString kLoopStatusPlaylist = QStringLiteral("Playlist");
```

Comment:
Shouldn't these be translated?

Reply:
no, those are status flags for the dbus api

Reply:
Okay. It would help to add a comment to clarify.